src/utilities/Utilities.jl

```diff
@@ -12,16 +12,10 @@ export  vectorize,
         Sample,
         Chain,
         init,
-        vectorize,
-        logpdf_binomial_logit,
```

Comment:
Are these not exported any more?

Reply:
It seems these are only for internal uses. Any reason for exporting them?

Reply:
No reason I can see.